### DISCUSSION TITLE
Update System and keep the data stored on EEPROM

### DISCUSSION THREAD

**Discussion by charmed12** [rauc/meta-rauc#214] [Q&A]
Hello , 
Hi, i have two system(s) on my board : system A and system B. The system B updates the first one using the yocto meta-rauc. 
I want to write some data in EEPROM through the system A only once. So if i update A the data in EEPROM , it dosen't erase.
Have you any idea, how it's done? 
Thank you.

**Comment by ejoerns** [comment]
HI @charmed12 

Not sure if I understand your question correctly. You want to write an EEPROM only on the very first boot ever? Thus probably just check if data is already in EEPROM and write if missing.

Or do you want do write data on each update? Is this a symmetric setup? Or do you really only write from B to A and thus more ore less have one full an one recovery system?
